rotkehlchen/exchanges/binance.py

```diff
@@ -589,6 +607,184 @@ def _query_lending_balances(
 
         return balances
 
+    def _api_query_list_paginated_within_time_delta(
+            self,
+            start_ts: Timestamp,
+            end_ts: Timestamp,
+            time_delta: int,
+            api_type: BINANCE_API_TYPE,
+            method: Literal['lending/union/interestHistory'],
+            additional_options: Optional[dict] = None,
+    ) -> list[dict[str, Any]]:
+        """Request via `_api_query_list_within_time_delta` using `current` param for pagination"""
+        data = []
```

Comment:
This can raise:

```
         - RemoteError
         - BinancePermissionError
```

make sure to state it in the docs of the functions that use this private method or here too and catch it where needed